configs/prod/pools.json

```diff
@@ -81,9 +81,11 @@
             "legacy_ids": [
                 "ETH-SQUIGGLEDAO"
             ],
+            "otc": false,
             "pool_name": "Grail NFTs",
             "token_decimals": 18,
             "token_symbol": "WETH",
+            "underwriting_slug": "grails",
```

Comment:
we should change this to `main` for now, since the underwriting service doesn't support the grails pools for now and it sends empty objects

Reply:
actually, we should keep it as `grails`, since the code wouldn't handle the change as intended

I didn't delete the comment for the sake of history